flatpack/src/test/java/net/sf/flatpack/delim/csv/CsvParserTest.java

```diff
@@ -69,6 +69,7 @@ public void testCsvWithWrongPzMap() {
      */
     public void testCsvDocumentWithMultilineString() {
         final String testCsv =
+            "col1,col2,col3,col4,col5,col6,col7" + System.lineSeparator() +
```

Comment:
sorry last time I made a mistake in this test, I missed column titles and verification was a bit incorrect. Now the test is fine and can replicate the issue. Changes in the parser solves it